.github/workflows/lokalise-automerger.yml

```
@@ -0,0 +1,71 @@
+name: Lokalise PR validation, auto approve and merge
+on:
+  pull_request:
+    branches:
+      - master
+      - develop
+    paths:
+      - 'src/locale/*.json'
+jobs:
+  autocheckandapprove:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Remove label
+        # More handy functions: https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#startswith
+        if: github.actor == 'VWSCoronaDashboard' && startsWith(github.head_ref, 'lokalise-') == true
+        uses: buildsville/add-remove-label@v1
+        with:
+          token: ${{ github.token }}
+          label: lokalise
+          type: remove
+
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        if: github.actor == 'VWSCoronaDashboard' && startsWith(github.head_ref, 'lokalise-') == true
+
+      - name: yarn install
+        if: github.actor == 'VWSCoronaDashboard' && startsWith(github.head_ref, 'lokalise-') == true
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: install
+
+      - name: EN locale validation
+        if: github.actor == 'VWSCoronaDashboard' && startsWith(github.head_ref, 'lokalise-') == true
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: validate-single locale en.json
+
+      - name: NL locale validation
+        if: github.actor == 'VWSCoronaDashboard' && startsWith(github.head_ref, 'lokalise-') == true
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: validate-single locale nl.json
+
+      - name: Add label
+        uses: buildsville/add-remove-label@v1
+        if: github.actor == 'VWSCoronaDashboard' && startsWith(github.head_ref, 'lokalise-') == true
+        with:
+          token: ${{ github.token }}
+          label: lokalise
+          type: add
+
+      - name: Auto approve
+        uses: hmarr/auto-approve-action@v2.0.0
+        if: github.actor == 'VWSCoronaDashboard' && startsWith(github.head_ref, 'lokalise-') == true
+        with:
+          github-token: ${{ github.token }}
+  automerge:
```

A new line would be nice over here
```suggestion

  automerge:
```